scripts/image_builder.sh

```diff
@@ -51,7 +51,7 @@ exit 1
 # Image file to be created:
 IMAGE="container.img"
 # Image contents source folder
-IMG_SIZE=${IMG_SIZE:-50M}
+IMG_SIZE=${IMG_SIZE:-80M}
```

Comment:
The `README.md` still mentions `50M` as the default so can you add an update for that too? It might be an idea to add a `make settings` rule or something to avoid having to update the docs every time we tweak this? That way, we can document, "run `make settings` to see what settings will be used to create the rootfs/image".

Reply:
I like the idea!, working on it.

Reply:
Hi @jodh-intel  updated docs, and captured #34